pkgs/games/mudlet/default.nix

```diff
@@ -98,7 +123,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Crossplatform mud client";
     homepage = "https://www.mudlet.org/";
-    maintainers = [ maintainers.wyvie maintainers.pstn ];
+    maintainers = [ maintainers.wyvie maintainers.pstn maintainers.cpu ];
```

Comment:
```suggestion
    maintainers = [ maintainers.wyvie maintainers.pstn maintainers.cpu ];
    maintainers = with maintainers; [ wyvie pstn cpu ];
```

Reply:
Fixed as of ae5ed8ce226db3913adf7b1107487f53bd8c69da.